resources/scss/componets/17_contact.scss

```diff
@@ -0,0 +1,56 @@
+.contact-section {
+    .bg-img {
+        background-size: cover;
+        background-repeat: no-repeat;
+
+        .info {
+            padding: 100px 5%;
+            position: relative;
+            z-index: 3;
+
+            .about {
+                margin-bottom: 50px;
+
+                h3 {
```

Comment:
do not use direct element tag in css
set class name like .title , .sub-title , .slogal , .description etc. like these

Reply:
Fixed